sr_description/mujoco_models/robot_ur10_hand_e_model.xml

```diff
@@ -0,0 +1,199 @@
+                <geom type="mesh" rgba="0.7 0.7 0.7 1" mesh="upperarm_top_link" pos="0 0 0" quat="1 0 0 0"/>
+                <body name="ra_forearm_link" pos="0 -0.1719 0.612">
+                    <inertial pos="0 0 0.28615" mass="3.87" diaginertia="0.11107 0.11107 0.0108844" />
+                    <joint name="ra_elbow_joint" pos="0 0 0" axis="0 1 0" limited="false" range="-3.14159 3.14159" />
+                    <geom type="mesh" rgba="0.7 0.7 0.7 1" mesh="forearm_bottom_link" />
+                    <geom type="mesh" rgba="0.7 0.7 0.7 1" mesh="forearm_body_link" pos="0 0 0" quat="1 0 0 0"/>
+                    <geom type="mesh" rgba="0.7 0.7 0.7 1" mesh="forearm_top_link" pos="0 0 0" quat="1 0 0 0"/>
+                    <body name="ra_wrist_1_link" pos="0 0 0.5723" quat="0.707107 0 0.707107 0">
+                        <inertial pos="0 0 0" quat="0.5 0.5 -0.5 0.5" mass="1.96" diaginertia="0.0055125 0.00510825 0.00510825" />
+                        <joint name="ra_wrist_1_joint" pos="0 0 0" axis="0 1 0" limited="false" range="-6.28319 6.28319" />
+                        <geom type="mesh" rgba="0.7 0.7 0.7 1" mesh="wrist1_base" />
+                        <geom type="mesh" rgba="0.7 0.7 0.7 1" mesh="wrist1_link" pos="0 0 0" quat="1 0 0 0" />
+                        <body name="ra_wrist_2_link" pos="0 0.1149 0">
+                            <inertial pos="0 0 0" quat="0.5 0.5 -0.5 0.5" mass="1.96" diaginertia="0.0055125 0.00510825 0.00510825" />
+                            <joint name="ra_wrist_2_joint" pos="0 0 0" axis="0 0 1" limited="false" range="-6.28319 6.28319" />
+                            <geom type="mesh" rgba="0.7 0.7 0.7 1" mesh="wrist2_base" />
+                            <geom type="mesh" rgba="0.7 0.7 0.7 1" mesh="wrist2_link" pos="0 0 0"/>
+                            <body name="ra_wrist_3_link" pos="0 0 0.1157">
+                                <inertial pos="0 0 0" quat="0.5 0.5 -0.5 0.5" mass="0.202" diaginertia="0.000568125 0.000526462 0.000526462" />
+                                <joint name="ra_wrist_3_joint" pos="0 0 0" axis="0 1 0" limited="false" range="-6.28319 6.28319" />
+                                <geom type="mesh" rgba="0.7 0.7 0.7 1" mesh="wrist3" />
+                                <body name="ra_ee_link" pos="0 0.0922 0" quat="0.707107 0 0 0.707107">
+                                    <inertial pos="-0.01 0 0" mass="0.001" diaginertia="1.66667e-08 1.66667e-08 1.66667e-08" />
+                                    <geom size="0.005 0.005 0.005" pos="-0.01 0 0" type="box" />
+                                        <body name="ra_arm_to_hand" euler="1.57079632679 1.57079632679 0">
+                                        <inertial pos="-0.01 0 0" mass="0.001" diaginertia="1.66667e-08 1.66667e-08 1.66667e-08" />
+                                        <geom size="0.005 0.005 0.005" pos="0.012 0.0 0.0" type="box"  />
+                                            <body name="rh_forearm">
```

Comment:
Re-defining the hand here rather than including it from the existing hand definition is going to make future hand changes harder to maintain